pages/linux/kscreen-console.md

```diff
@@ -0,0 +1,32 @@
+# kscreen-console
+
+> KScreen Console is a CLI tool to query KScreen's status.
```

Comment:
Perhaps this will read better?
```suggestion
> Command line tool to query KScreen's status.
```

Reply:
Just hijacking this so I can add a hyphen!

```suggestion
> Command-line tool to query KScreen's status.
```

Reply:
@SethFalco FYI, you could've just edited @sbrl's suggestion (like I did at the exact same time that you added a new suggestion 🙂)

Reply:
I did consider that, but I'm always hesitant to edit the comments of others.
Feels a bit weird to edit the user-generated content of another user.

I'll keep it in mind for future if it's something minor like this, though.

Reply:
Generally I'm hesitant too. If I do edit others' comments, I always leave a comment of my own.

Any particular reason for `Command-line` over `Command line`? I think it looks odd and weird with a hyphen.

Reply:
Statistics:

 - `command line`: 589 times found in 323 pages
 - `command-line`: 218 times found in 211 pages

Reply:
Context:
https://github.com/tldr-pages/tldr/pull/5848#discussion_r635310449

Reply:
Because `command-line` is really almost a compound, one-word adjective.

Reply:
I got those numbers with `tldr --search` using the tldr bash client. Hrm, I guess we can go with `command-line` then. But it still looks very wrong to me.

Reply:
I think `command-line` is easier to read. But that's also, because in German you'd write this as a single word.